auth.go

```diff
@@ -62,6 +67,42 @@ func (a *Auth) CheckPermission(scope, userID string) error {
 	return ErrNotAllowed
 }
 
+// GetUsers returns a list of userIDs having one or more permission scopes
+func (a *Auth) GetUsers() ([]string, error) {
+	keys, err := a.store.Keys()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to load permissions")
+	}
+
+	a.logger.Debug("Retrieving all userIDs")
```

Comment:
We should probably add this log message to the top of the function before we make the call to the store. This can help if for some reason it hangs and we want to know what it was trying to do.

Reply:
updated in https://github.com/go-joe/joe/pull/28/commits/490bd8b858d0eb5ce7b2aab912a68077095a1acb